src/pages/RFOX/hooks/useEpochHistoryQuery.ts

```diff
@@ -13,7 +14,10 @@ import {
 
 type EpochHistoryQueryKey = ['epochHistory']
 
-const RFOX_FIRST_EPOCH_START_TIMESTAMP = BigInt(dayjs('2024-07-01T00:00:00Z').unix())
+// TEMP: This is a temporary hack to allow us to mock the rewards tx history for RFOX
+const RFOX_FIRST_EPOCH_START_TIMESTAMP = getConfig().REACT_APP_FEATURE_RFOX_MOCK_REWARDS_TX_HISTORY
+  ? BigInt(dayjs('2024-07-01T00:00:00Z').subtract(3, 'months').startOf('month').unix())
+  : BigInt(dayjs('2024-07-01T00:00:00Z').unix())
```

Comment:
👀

Reply:
@woodenfurniture I was able to repro the lifetime rewards bug spotted by @NeOMakinG, goes away after reverting this guy

Reply:
Right, so this flag is for testing this PR only so does that mean not blocked?

Reply:
Super weird but I'm still rugged with the flag off, could you give it a try? 🙏

Reply:
i.e these guys
![image](https://github.com/shapeshift/web/assets/125113430/4108429d-c0aa-49b0-b06b-8a8b8b0d075e)

Reply:
> Super weird but I'm still rugged with the flag off, could you give it a try please? 🙏
> 
> ![image](https://private-user-images.githubusercontent.com/14963751/346739949-f9f2d0e9-5677-4b1d-b076-e3bd3eed8d62.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0ODA1MjQsIm5iZiI6MTcyMDQ4MDIyNCwicGF0aCI6Ii8xNDk2Mzc1MS8zNDY3Mzk5NDktZjlmMmQwZTktNTY3Ny00YjFkLWIwNzYtZTNiZDNlZWQ4ZDYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDIzMTAyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRlM2I0NTMzMTk4OTM0MDJjZWZmMGVlYTc0NDExNTJhYTllMDAyZDk3MWJlOWI3MWRhYTY1OTcxMWFiYzJmODUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.f-bSQzX_RrOFBtjChMcIKft7oihMMlZWqLsQQglUEj8)

If i understand correctly, there are no rewards yet as it's the first epoch so should be empty

Reply:
Also noting the dev env has a different staking contract to prod

Reply:
confirmed this is just an artefact of the test flags
![image](https://github.com/shapeshift/web/assets/125113430/0f497e8b-71f9-4e99-b120-d95669654a33)

Reply:
We're merging as-is for now with `lifetime rewards` broken with the flag on, noting a full re-work of all dashboard queries is needed to work without an archive node, i.e all of this will be rewritten anyway